.gitignore

```diff
@@ -5,12 +5,12 @@ result-original
 
 # IDEs
 .idea/
+.vscode/
```

Comment:
.vscode/settings.json needs to be here so this needs to be more specific or removed